power_grid_model_c/power_grid_model/include/power_grid_model/enum.hpp

```diff
@@ -94,6 +94,8 @@ enum class FaultPhase : IntS {
     nan = na_IntS
 };
 
+enum class ShortCircuitVoltageScaling : IntS { minimum = 0, maximum = 1 };
```

Comment:
maybe not for this time cause it's a bit late now but in the future it is probably a good idea to split this type of change into 2 PRs:

1. refactor the existing behaviour (does not require new tests; no new functionality)
2. actually implement the new behaviour (requires new tests)

doing it that way helps isolate the actual implementation details of the new behaviour (making it easier to review, debug and explain), as well as makes the confidence higher that no existing functionality breaks at any given time (because it generally implies that **either** code changes are small but test changes are big, **or** code changes are big but test changes are small - and only very rarely both